applications/Covid Tracker/readme

```diff
@@ -0,0 +1,4 @@
+I have just quickly created how the website might look. 
+This is only for practicing html and css. Feel free to join in, in building the application. 
+Please mention, what changes are you making though! 
+:)
```

Comment:
If you change the name of this file to `README.md`, it will display on GitHub when someone navigates to this folder.

Reply:
I'll do this